tests/_support/constants/Timeouts.js

```diff
@@ -0,0 +1,5 @@
+const Timeouts = {
+  SERVER_RESPONSE_DELAY: 200
```

Comment:
Let's get a comment here:
`// 200 seems to be enough time to make an assessment before the server responds`